Makefile

```diff
@@ -57,10 +45,9 @@ packages: rpm srpm
 
 # Python packaging
 wheel:
-	python setup.py bdist_wheel
-	python3 setup.py bdist_wheel
+	hatch build
```

Comment:
```suggestion
	hatch build -t wheel
```

Reply:
Hmm, if that's the case I should also make an `sdist` target and make `upload` depend on both?